cli/ballerina-cli/src/main/resources/cli-help/ballerina-asyncapi.help

```diff
@@ -0,0 +1,36 @@
+NAME
+      Generate a Ballerina listener from an AsyncAPI contract.
+
+SYNOPSIS
+       bal asyncapi [-i | --input] <asyncapi-contract-file-path> [-o | --output] <output-location>
+
+
+DESCRIPTION
+       Generate a Ballerina source from the given AsyncAPI definition file. This will generate four Ballerina files.
+        1. data_types.bal - This file contains all the data types in the listener
+        2. service_types.bal - This file contains all the service types relevant to the generated listener
+        3. listener.bal - This file contains the http listener which listens to the relevent third party service
+        4. dispacther_service.bal - This file contains the event dispatching logic
+    
+       The generated Ballerina sources will be written into the provided output location.
+
+       Note: This is an experimental feature, which supports only a limited
+       set of functionalities.
+
+
+OPTIONS
+        -i, --input <asyncapi-contract-file-path>
+            This is a mandatory input. The listener will be generated according to the given AsyncAPI contract. 
+            This generated listener can be used as a code template to start the listener implementation.
+
+        -o, --output <output>
+            Location of the generated Ballerina source code. If this path is not specified,
+            the output will be written to the same directory from which the command is run.
+
+
+EXAMPLES
+       Generate a Ballerina listener using a `hello.yaml` AsyncAPI contract.
+           $ bal asyncapi -i hello.yaml
+
+       Generate a Ballerina listener using a `hello.yaml` AsyncAPI contract with an output path
```

Comment:
```suggestion
       Generate a Ballerina listener using a `hello.yaml` AsyncAPI contract with an output path.
```

Reply:
Thanks a lot for all the above suggestions. Fixed them